infra/base-images/base-runner/coverage

```diff
@@ -111,8 +111,22 @@ function run_fuzz_target {
   fi
 }
 
+export GOPATH=/out/root/go
```

Comment:
if we decide to install Go in the base-image (the comment above), we should set the `GOPATH` there too. Then, here we will do `GOPATH=$OUT/$GOPATH` to keep things less fragile

Reply:
Waiting for the decision about where to put go, otherwise ok

Reply:
Let me ping @oliverchang once again, as GitHub reviews are terrible. Oliver, we need to have Go installed in both the builder and the runner images. Would you be concerned if we install Go only once in the `base-image` instead of `base-builder` and `base-runner`?